recipes/inspyred/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{%set name = "inspyred" %}
+{%set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/5e/f3/3d5f05b3be6336691277bcf088197caca789d0cf02c4a7fb2c3f21b021ab/{{ name }}-{{ version }}.tar.gz
+  md5: ce51b599f5682ec526c97412f3ebaf45
+  patches:
+    - without-ez_setup.patch  # just use setuptools
+    - 0001-Add-README.md.patch  # required from setup.py but missing from pypi archive
```

Comment:
Why do we need this? Also please raise an issue upstream to have `README.md` added to `MANIFEST.in`.